### DISCUSSION TITLE
Frequency visible on QSO tab

### DISCUSSION THREAD

**Discussion by gx1400** [magicbug/Cloudlog#2101] [Ideas / Feature Requests]
When using CloudlogCAT or similar API link to radio, it would be nice to see frequency on the main QSO tab to be able to ensure radio is still linked and that tge frequency is accurate before logging.

**Comment by phl0** [comment]
Patch proposal in https://github.com/magicbug/Cloudlog/pull/2108. You are invited to test and comment :).

**Reply by gx1400** [reply]
I will pull the PR and try to test sometime this week

**Reply by phl0** [reply]
Screenshot attached to the PR :-)

**Reply by m0urs** [reply]
Looks good to me. Thank you!

maybe we could round the frequency to a max of six digits after the decimal point?

![image](https://user-images.githubusercontent.com/2588362/236292167-b34c2a6f-20ba-43ad-8e98-8a71d6ac30a7.png)

**Reply by R1BLH** [reply]
Looks good! Thank you!

![2023-05-16_21-19-54](https://github.com/magicbug/Cloudlog/assets/16667173/df90949c-f2c0-437d-bd4d-a8a2e5724989)
It would be nice to see modulation mode aside frequency as well.

**Comment by gx1400** [comment]
I have been unable to test and probably will not be able to do so, but I look forward to seeing this merged in the future!

**Reply by phl0** [reply]
Well it is in current master branch (v 2.4.2)

**Reply by gx1400** [reply]
I noticed that about 10 minutes after I posted. I'm still working on getting an antenna in the attic so I dont have a base station setup yet.

Thank you for the note.  I imagine this issue could be closed, any future feedback can be a new issue

**Comment by sipsmi** [comment]
![image](https://github.com/magicbug/Cloudlog/assets/22954170/c69416b8-9914-49db-a826-b26daea97145)
Very nice feature.
I don't need the mode though its always CW :-)

**Reply by phl0** [reply]
Then check the program u use for providing the API with info. Does it deliver mode at all?

**Reply by sipsmi** [reply]
Sorry, I meant I only operate on CW not that it does not pick the mode up from the API call 😁 I guess a multimode transceiver is wasted on me unless you count CW-u and CW-l (which I had to hack the cloudlogQt to handle BTW

**Reply by phl0** [reply]
Oh right. Sorry for the misunderstanding. So I take it as fixed then.
